vividus-bdd-engine/src/main/java/org/vividus/bdd/spring/ExtendedConfiguration.java

```diff
@@ -71,6 +73,7 @@ public void init() throws IOException
         customTableTransformers.forEach(transformers::useTransformer);
         useStepMonitor(new DelegatingStepMonitor(stepMonitors));
         useStoryControls(storyControls);
+        useParameterControls(parameterControls);
```

Comment:
it should be placed between before  `useParameterConverters` since `ParameterConvertersDecorator` constructor uses `ParameterControls`